hass_client/client.py

```diff
@@ -170,35 +181,44 @@ async def call_service(
             msg["service_data"] = service_data
         if target:
             msg["target"] = target
-        return await self.send_command(msg)
+        return await self.send_retryable_command(msg)
 
     async def get_states(self) -> list[State]:
         """Get dump of the current states within Home Assistant."""
-        return await self.send_command("get_states")
+        return await self.send_retryable_command("get_states")
 
     async def get_config(self) -> list[Config]:
         """Get dump of the current config in Home Assistant."""
-        return await self.send_command("get_states")
+        return await self.send_retryable_command("get_config")
 
     async def get_services(self) -> dict[str, dict[str, Any]]:
         """Get dump of the current services in Home Assistant."""
-        return await self.send_command("get_services")
+        return await self.send_retryable_command("get_services")
 
     async def get_area_registry(self) -> list[Area]:
         """Get Area Registry."""
-        return await self.send_command("config/area_registry/list")
+        return await self.send_retryable_command("config/area_registry/list")
 
     async def get_device_registry(self) -> list[Device]:
         """Get Device Registry."""
-        return await self.send_command("config/device_registry/list")
+        return await self.send_retryable_command("config/device_registry/list")
 
     async def get_entity_registry(self) -> list[Entity]:
         """Get Entity Registry."""
-        return await self.send_command("config/entity_registry/list")
+        return await self.send_retryable_command("config/entity_registry/list")
 
     async def get_entity_registry_entry(self, entity_id: str) -> Entity:
         """Get single entry from Entity Registry."""
-        return await self.send_command("config/entity_registry/get", entity_id=entity_id)
+        return await self.send_retryable_command("config/entity_registry/get", entity_id=entity_id)
+
+    async def send_retryable_command(self, command: str, **kwargs: dict[str, Any]) -> CommandResultData:
```

Comment:
I'm not sure if I'm a fan of this pattern. Why not introduce an argument to specify if the command may be retried ?
Also, I think it would be good to specify a number of retries. So for example introduce an argument "retries:int =3" and thus by default retry a command 3 times on common connection errors